Build.fsx

```diff
@@ -13,7 +13,7 @@ let nuGetOutputFolder = "NuGetPackages"
 let solutionsToBuild = !! "Src/*.sln"
 let signKeyPath = FullName "Src/AutoFixture.snk"
 
-type GitVersion = { apiVersion:string; nugetVersion:string; }
+type GitVersion = { apiVersion:string; nugetVersion:string }
```

Comment:
I think this whole commit should be squashed into 6e6dad9 as it's unclear what it does on its own. What's the "_optimization_" here?